Dockerfile-localnet

```diff
@@ -0,0 +1,57 @@
+FROM golang:1.20-alpine3.18
```

Comment:
What is the purpose of this one compared to `Dockerfile`? Can we add comment at the top of the file to explain the context?
Asking because we used to have too many unnecesary dockerfile and scripts, so it is good to have access to context when performing cleaning up